master_startup.py

```diff
@@ -10,6 +10,8 @@
 import BMP280 as barometer
 import RPi.GPIO as io
 from imu_utils import IMU
+from subprocess import call
```

Comment:
I'd just say `subprocess.call()`. It's like `using namespace dtf;` in C++...

Reply:
Ah, sure-- I was only familiar with this, there's an easy fix.